Update readme style (#60)
substrate-developer-hub/substrate-front-end-template: README.md

# Substrate Front End Template

This template allows you to create a front-end application that connects to a
[Substrate](https://github.com/paritytech/substrate) node back-end with minimal
configuration. To learn about Substrate itself, visit the
[Substrate Developer Hub](https://substrate.dev).

The template is built with [Create React App](https://github.com/facebook/create-react-app)
and [Polkadot js API](https://polkadot.js.org/api/). Familiarity with these tools
will be helpful, but the template strives to be self-explanatory. To learn how
this template was built, visit the
[Substrate Front-End Tutorial](https://substrate.dev/docs/en/tutorials/substrate-front-end/).

## Using The Template

### Installation

The code can be installed using [git](https://git-scm.com/) and [yarn](https://yarnpkg.com/).
```bash
# Clone the repository
git clone https://substrate-developer-hub/substrate-front-end-template.git
cd ./substrate-front-end-template
```

## Usage
You can start the template in development mode to connect to a locally running node
```bash
yarn start
```

You can also start it in production mode...

> TODO @jimmychu What are the details about this again?

Reviewer: `yarn build` triggers the production settings to kick in.

Author: Added those details. I'm open to suggestions on how to format that part.

> ## Configuration

Reviewer: ```suggestion
## Production Configuration
```

The template's configuration is in three files in the `src/config` directory.
* `common.json` affects both development and production environments.
* `development.json` affects only the development environment with higher
precedence than `common.json`.
* `production.json` affects only the production environment with higher
precedence than `common.json`.
TODO @jimmychu what's `test.json`? Can you please write about it or remove it?

More on
[React environment variables](https://create-react-app.dev/docs/adding-custom-environment-variables).

> When writing and deploying your own front end, you will likely change:

Reviewer: ```suggestion
When writing and deploying your own front end, you should configure:
```

and [Polkadot js API](https://polkadot.js.org/api/). Familiarity with these tools
will be helpful, but the template strives to be self-explanatory. To learn how
this template was built, visit the
[Substrate Front-End Tutorial](https://substrate.dev/docs/en/tutorials/substrate-front-end/).

## Using The Template

### Installation

The code can be installed using [git](https://git-scm.com/) and [yarn](https://yarnpkg.com/).
```bash
# Clone the repository
git clone https://substrate-developer-hub/substrate-front-end-template.git
cd ./substrate-front-end-template
```

## Usage
You can start the template in development mode to connect to a locally running node
```bash
yarn start
```

You can also build the app in production mode,
```bash
yarn build
```
and open `build/index.html` in your favorite browser.

## Configuration

The template's configuration is in three files in the `src/config` directory.
* `common.json` affects both development and production environments.
* `development.json` affects only the development environment with higher
precedence than `common.json`.
* `production.json` affects only the production environment with higher
precedence than `common.json`.

> TODO @jimmychu what's `test.json`? Can you please write about it or remove it?

Reviewer 2: `test.json` is used when running `yarn test` in the CI workflow.

Author: Thanks for adding the info :)
